### PR TITLE
feat: Automating build deploys to CDN

### LINKED ISSUE #402: Automating build deploys  to CDN
Refers: https://github.com/freeCodeCamp/testable-projects-fcc/pull/397#issuecomment-401158124
Refers: https://github.com/freeCodeCamp/cdn/pull/2#issuecomment-401244723

### PR DESCRIPTION
**Pull request by ValeraS**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [ ] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.

- [x] Closes currently open issue: Closes #402

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+if [ -n "$CDN_API_KEY" ]; then
+  git clone https://github.com/freeCodeCamp/cdn.git
+
+  # Copy bundle.js to local CDN repo
+  cp --parents build/testable-projects-fcc/v1/bundle.js ../cdn/
```
[the remark this answers is not in the document]
Comment (tbushman) [reply deploy.sh]: ... * looks up `cp --parents` *  
I was wondering -- would that overwrite `../cdn/build/index.html` or other files in the existing structure?

[the remark this answers is not in the document]
Comment (ValeraS) [reply deploy.sh]: @tbushman  `cp --parents` - copy a file and, if needed, makes dirs. All other files will be the same

[the remark this answers is not in the document]
Comment (ValeraS) [reply deploy.sh]: @tbushman We can write this
```
cp -r --parents build/* ../cdn/
```
or
```
cp -r --parents build/testable-projects-fcc/* ../cdn/
```
to take into account future changes

**Comment by ValeraS** [comment]
@tchaffee Thank you for your patience. I appreciate it.
I used this PR only to share my thoughts on this topic.

Excuse me for disturb, can you answer my question here https://github.com/freeCodeCamp/testable-projects-fcc/issues/512#issuecomment-414065403?

**Comment by tchaffee** [comment]
@ValeraS I just want to make sure you don't duplicate effort. In this case, I think you are already way ahead of me so as soon as I get a chance I'll review this PR. Do you think it works as-is and is ready for merge? Or do you still need to work on it and wanted to discuss it?

**Comment by ValeraS** [comment]
@tchaffee I think it's ready, but a fresh look will be useful.

**Comment by tchaffee** [comment]
@ValeraS I did a final review and everything looks good. Can you please rebase over master to get rid of the merge conflicts? As soon as you do that I'll merge this. Thanks!

**Comment by ValeraS** [comment]
@tchaffee It's done. You can merge it.

**Pull request merged**